xml/System.IO/Stream.xml

```diff
@@ -1076,6 +1076,8 @@
 ## Remarks  
  Override `Flush` on streams that implement a buffer. Use this method to move any information from an underlying buffer to its destination, clear the buffer, or both. Depending upon the state of the object, you might have to modify the current position within the stream (for example, if the underlying stream supports seeking). For additional information see <xref:System.IO.Stream.CanSeek%2A>.  
   
+ In a class derived from <xref:System.IO.Stream> that doesn't support writing, <xref:System.IO.Stream.Flush%2A> is typically implemented as an empty method to ensure full compatibility with other <xref:System.IO.Stream> types since it's valid to flush a read-only stream.  
```

Comment:
Same here: comma before "since"

Also note that this results in a one-sentence paragraph, which isn't ideal. However, it's not easy to break this, and it doesn't seem to require any additional explanation. Let's not sweat it. It's not a huge giant big deal. :smile: